infer/src/base/Config.ml

```diff
@@ -497,6 +498,14 @@ and bootclasspath =
     ~exes:CLOpt.[Toplevel; Java]
     "Specify the Java bootclasspath"
 
+and bo_debug =
+  CLOpt.mk_int ~default:0 ~long:"bo-debug"
+    ~exes:CLOpt.[Toplevel] "Debug mode for buffer-overrun checker"
+
+and bo_filtering =
```

Comment:
Oops, I pointed you to the wrong place. You are right that this code is useless; good catch. I meant to point you to [here](https://github.com/facebook/infer/blob/master/infer/src/backend/InferPrint.re#L344).